nitransforms/io/afni.py

```diff
@@ -193,6 +193,17 @@ def from_image(cls, imgobj):
 
         return imgobj.__class__(field, imgobj.affine, hdr)
 
+    @classmethod
+    def to_image(cls, imgobj):
+        """Export a displacements field from a nibabel object."""
+
+        hdr = imgobj.header.copy()
+
+        warp_data = imgobj.get_fdata().reshape(imgobj.shape[:3] + (1, imgobj.shape[-1]))
```

Comment:
@effigies, shouldn't the dataobj be reshaped directly here without get_fdata?

Reply:
If we were only reshaping, sure. But since we need to flip signs in the next line, you do need the loaded array.